doc/book/controllers.md

```diff
@@ -280,9 +280,9 @@ extending class.
 ## AbstractConsoleController
 
 > To use  it, you need to install zend-mvc-console via
```

Comment:
Can you extend the sentence to clarify that "zend-mvc-console" is a component and it is required to work with the console features. Also a link to component documentation would be good.